### PR TITLE
Added pretty representation of ProjectSchema for IPython

### LINKED ISSUE #314: project.detect_schema() should have a pretty representation in IPython
### Feature description
In #127, we added an IPython representation for `JobsCursor` objects. I would like the same kind of behavior for `ProjectSchema` objects, with a pretty format that looks similar to the output of `signac schema`.

### Proposed solution

Add a method `_repr_html_` to `ProjectSchema`. Refer to `JobsCursor` and `Project` classes for examples.

### PR DESCRIPTION
**Pull request by vishav1771**
## Description

Added `_repr_html_` method in  `ProjectSchema`
## Motivation and Context
Resolves #314 

## Types of Changes

- [ ] Documentation update
- [ ] Bug fix
- [x] New feature
- [ ] Breaking change<sup>1</sup>

<sup>1</sup>The change breaks (or has the potential to break) existing functionality.

## Checklist:

- [x] I am familiar with the [**Contributing Guidelines**](https://github.com/glotzerlab/signac/blob/master/CONTRIBUTING.md).
- [x] I agree with the terms of the [**Contributor Agreement**](https://github.com/glotzerlab/signac/blob/master/ContributorAgreement.md).
- [x] My name is on the [list of contributors](https://github.com/glotzerlab/signac/blob/master/contributors.yaml).
- [x] My code follows the [code style guideline](https://github.com/glotzerlab/signac/blob/master/CONTRIBUTING.md#code-style) of this project.
- [x] The changes introduced by this pull request are covered by existing or newly introduced tests.

If necessary:
- [ ] I have updated the API documentation as part of the package doc-strings.
- [ ] I have created a separate pull request to update the [framework documentation](https://docs.signac.io/) on [signac-docs](https://github.com/glotzerlab/signac-docs) and linked it here.
- [x] I have updated the [changelog](https://github.com/glotzerlab/signac/blob/master/changelog.txt) and added all related issue and pull request numbers for future reference (if applicable). See example below.

Example for a changelog entry: `Fix issue with launching rockets to the moon (#101, #212).`

### CODE REVIEW

File: signac/contrib/schema.py [hunk 1]
```diff
@@ -136,6 +138,15 @@ def __str__(self):
     def __repr__(self):
         return "{}(<len={}>)".format(type(self).__name__, len(self))
 
+    def _repr_html_(self):
+        html = "{<br>"
```
Comment (b-butler) [inline signac/contrib/schema.py:142]: I think we should still include the type name for the project before the opening bracket.

File: signac/contrib/schema.py [hunk 2]
```diff
@@ -136,6 +138,15 @@ def __str__(self):
...
+        for key in sorted(self._schema):
+            values = self._schema[key]
+            if values:
+                html += "<pre>  '{}': '{}',</pre>".format(key, self._fmt_values(values, 5))
+        html += "}"
+        return html
```
Comment (bdice) [inline signac/contrib/schema.py]: I think Brandon has a good recommendation above. @vishav1771 It looks like this PR is re-creating the same formatting logic that is in `__str__`. I suggest the following code snippet -- I'm happy with this output (screenshot below).
```suggestion
        import html
        html_repr = "<strong>" + html.escape(repr(self)) + "</strong>"
        html_repr += "<pre>" + str(self) + "</pre>"
        return html_repr
```
image

**Comment by bdice** [comment]
@vishav1771 One more thing - you might want to update your remote's `master` branch so that it's up-to-date with the `glotzerlab:master` branch. I think it might be somewhat behind.

**Comment by bdice** [comment]
I also reverted the changes to `_fmt_values` because those changes are no longer necessary with my changes to `_repr_html_`.

**Pull request merged**